configs/disco/README.md

```diff
@@ -0,0 +1,149 @@
+# Disco Diffusion(Google Colab)
+
+> [](<>)
+
+> **Task**: Text2Image
+
+<!-- [ALGORITHM] -->
+
+## Abstract
+
+<!-- [ABSTRACT] -->
+
+Disco Diffusion (DD) is a Google Colab Notebook which leverages an AI Image generating technique called CLIP-Guided Diffusion to allow you to create compelling and beautiful images from just text inputs. Created by Somnai, augmented by Gandamu, and building on the work of RiversHaveWings, nshepperd, and many others.
```

Comment:
```suggestion
Disco Diffusion (DD) is a Google Colab Notebook which leverages an AI Image generating technique called CLIP-Guided Diffusion to allow you to create compelling and beautiful images from text inputs. 

Created by Somnai, augmented by Gandamu, and building on the work of RiversHaveWings, nshepperd, and many others.
```

Reply:
Or move the contributors to the section `Credits`

Reply:
> Please DO provide UT to ensure basic correctness.

Sure.